packages/typespec-ts/src/modular/helpers/operationHelpers.ts

```diff
@@ -718,6 +719,8 @@ function deserializeResponseValue(
       return required
         ? `new Date(${restValue})`
         : `${restValue} !== undefined? new Date(${restValue}): undefined`;
+    case "combined":
+      return `${restValue} as any`;
```

Comment:
Temp workaround for this issue https://github.com/Azure/autorest.typescript/issues/1921